tests/test_service.py

```diff
@@ -412,6 +412,44 @@ def test_delete_existing_wishlist(self):
         )
         self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
 
+    def test_update_existing_wishlist(self):
+        """ Update an existing Wishlist """
+        # Create a wishlist to update
+        test_wishlist = WishlistFactory()
+        resp = self.app.post(
+            "/wishlists", json=test_wishlist.serialize(),
+            content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
+        # Update the wishlist
+        new_wishlist = resp.get_json()
+        new_wishlist["name"] = "devops"
+        resp = self.app.put(
+            "/wishlist/{}".format(new_wishlist["id"]),
+            json=new_wishlist,
+            content_type="application/json",
+        )
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        updated_wishlist = resp.get_json()
+        self.assertEqual(updated_wishlist["name"], "devops")
+
+    def test_update_non_existing_wishlist(self):
+        """ Update a non-existing Wishlist """
+        test_wishlist = WishlistFactory()
+        resp = self.app.put(
+            "/wishlists/{}".format(test_wishlist.id),
+            content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_204_NO_CONTENT)
```

Comment:
If we try to update a non-existing wishlist, should it not give a 404. 204 is no content used for delete I believe. Thoughts?

Reply:
Resolved.